README.md

```diff
@@ -74,16 +74,22 @@ Need:
 
 - Cake 2: <http://bakery.cakephp.org/articles/lorenzo/2011/10/16/cakephp_2_0_released>
 
+- Typo3 Flow 1: <http://en.wikipedia.org/wiki/TYPO3_Flow>
```

Comment:
@rajibmp what I was trying to say was rather than wikipedia link it will be nice to point to release post. Anyway @pmjones will say the last word.

And I do have played with flow :-) .